Project.toml

```diff
@@ -1,5 +1,7 @@
 name = "CloudWatchLogs"
 uuid = "c4c1e6a2-6bdd-52e0-b56d-1d4734724d2d"
+authors = ["Invenia Technical Computing"]
```

Comment:
Is our standard `Invenia Technical Computing` or `Invenia Technical Computing Corporation`?

Reply:
I'm not sure, the latter seems rather long.

Reply:
I opted to avoid adding that section to Nabla's Project.toml pending further clarification from The Officials.

Reply:
Standard is "Invenia Technical Computing Corporation" as recorded in InveniaTemplates.jl